CMakeLists.txt

```diff
@@ -79,6 +79,18 @@ else()
   set(MENDER_USE_BOOST_FS 0)
 endif()
 
+if (${PLATFORM} STREQUAL linux_x86)
```

Comment:
Sounds pretty much like: cmake_dependent_option to me: https://cmake.org/cmake/help/latest/module/CMakeDependentOption.html#module:CMakeDependentOption